.github/workflows/release_prep.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+# Set by GH actions, see
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+TAG=${GITHUB_REF_NAME}
+# The prefix is chosen to match what GitHub generates for source archives
+PREFIX="capnp-cpp-${TAG:1}"
+ARCHIVE="capnp-cpp-$TAG.tar.gz"
+git archive --format=tar --prefix=${PREFIX}/ ${TAG} | gzip > $ARCHIVE
```

Comment:
Effectively, this is the line that generates the `capnp-cpp-${VERSION}.tar.gz` archive